### DISCUSSION TITLE
Best ways to interact on live issues?

### DISCUSSION THREAD

**Discussion by papakpmartin** [weather-gov/api#467] [Q&A]
On #466 I learned from @scadergit that active issues with the responses and data that this API is providing should be emailed to NCO at nco.ops@noaa.gov, but I am wondering if there's any place to check on on status of things like outages, known issues, etc.

Emailing to NCO seems (so far for me) to be pretty one way communication, but if I'm seeing something happening “now” it would be useful to not spend time debugging my code if it’s just an API thing.

**Comment by JJKraw** [comment]
If it's something major, usually something is posted here:

  https://www.nco.ncep.noaa.gov/status/messages/

**Comment by jah1007** [comment]
Long term we are investigating solutions that could build user lists for the API. Mining our access logs is understandably cumbersome and not every user or application provided enough information in the User-Agent that we could contact them. The link @JJKraw provided will capture big outages. If you are unsure of the status you can reach out to the nco.ops email but also the Tier-2 (application support) team at idp-support@noaa.gov. @scadergit or @StephenClouse can we get this added where nco.ops is documented?
